spec/recipes/dd-agent_spec.rb

```diff
@@ -1008,23 +993,6 @@ def set_env_var(name, value)
 
   context 'agent 6' do
     describe 'the datadog-agent service' do
-      context 'on Amazon Linux < 2.0' do
```

Comment:
What about these removed tests for older platforms, or the change from Ubuntu 14->16? How can we keep these tested?

Reply:
These are only the unit tests and unfortunately Fauxhai doesn’t provide for older versions if you want to also test current versions.

However! The integration tests _can_ be run against older distro versions, and are in general a far more “real” set of tests.

Right now those are a smaller set of tests, but they can be expanded. You can look back at the history of this PR and its builds to see me wrestling with that, since quite a few things pass the unit tests, but fail when run in reality as an integration test.

Reply:
Thanks a lot for your explanation, that makes sense.